notebook/agentchat_microsoft_fabric.ipynb

```diff
@@ -44,31 +42,46 @@
     }
    },
    "source": [
-    "### Install AutoGen"
+    "### Example 0"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "id": "2",
-   "metadata": {},
-   "outputs": [],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
    "source": [
-    "%pip install \"pyautogen[retrievechat,lmm]>=0.2.28\" -q"
+    "#### Work with openai<1\n",
+    "\n",
+    "AutoGen can work with openai<1 in Microsoft Fabric. To access pre-built LLM endpoints with AutoGen, you can follow below example.\n",
+    "\n",
+    "This example can run in Fabric runtime 1.2+."
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "id": "3",
    "metadata": {
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
     "nteract": {
      "transient": {
       "deleting": false
      }
     }
    },
+   "outputs": [],
    "source": [
-    "### Set up config_list and llm_config"
+    "# pyautogen<=0.1.14 supports openai<1\n",
+    "%pip install \"pyautogen==0.1.14\" \"openai==0.28.1\" -q"
```

Comment:
are we not using the `autogen` package?

Reply:
Here it's pinning to an older version which only exists for pyautogen.

Reply:
Sorry for the late reply, yes that's correct